sqlobject/src/main/java/org/jdbi/v3/sqlobject/transaction/internal/TransactionDecorator.java

```diff
@@ -29,7 +30,7 @@
     public Handler decorateHandler(Handler base, Class<?> sqlObjectType, Method method) {
         final Transaction txnAnnotation = Stream.of(method, sqlObjectType)
                 .map(ae -> ae.getAnnotation(Transaction.class))
-                .filter(txn -> txn != null)
+                .filter(Objects::nonNull)
```

Comment:
this is one of those places where the explicit code is more readable than the function reference. Now I have to look up what "Objects::nonNull" does. The previous code was more readable.

Reply:
it's a matter of taste :)
personally I like method references